src/plugins/visualizations/public/vis_types/types.ts

```diff
@@ -41,13 +47,16 @@ export interface VisType<TVisParams = unknown> {
   readonly name: string;
   readonly title: string;
   readonly description?: string;
+  readonly note: string;
```

Comment:
:scroll: Could we please add a JSDoc description for those new parameters to explain how they behave? I have the feeling especially for `groupTitle` and `note` it's not self explanatory (at least I didn't get it without reading the remaining code).

Reply:
Sure I will add it to all params 😉

Reply:
For `note` specifically: do we really need to add this? I have the feeling we're cluttering the interface here a lot for one-time use-cases. Couldn't we instead make `description` be a react element instead? That way it would work fine just specifying a string for most cases, and the few visualizations that might need some italic part in there, can just return JSX containing it?